lib/redis-session-store.rb

```diff
@@ -128,6 +128,11 @@ def set_session(env, sid, session_data, options = nil)
   end
   alias write_session set_session
 
+  def get_expiry(env, options)
+    session_storage_options = options || env.fetch(ENV_SESSION_OPTIONS_KEY)
```

Comment:
Will `ENV_SESSION_OPTIONS_KEY` always be present? If not, then perhaps use `fetch` with a default value, such as `env.fetch(ENV_SESSION_OPTIONS_KEY, {})`? That way, if session_storage_options is nil, `session_storage_options[:ttl]` won't raise a NoMethodError.